### PR TITLE
Some changes

### PR DESCRIPTION
**Pull request by ice1000**
@imkiva

### CODE REVIEW

File: base/src/main/java/org/aya/core/term/Term.java [hunk 1]
```diff
@@ -122,15 +122,8 @@ public sealed interface Term extends AyaDocile, Restr.TermLike<Term> permits Cal
...
-          case IMIN -> PrimTerm.Mula.and(args.first().term(), args.last().term());
-          case IMAX -> PrimTerm.Mula.or(args.first().term(), args.last().term());
-          case INVOL -> PrimTerm.Mula.inv(args.first().term());
-          default -> {
-            if (args.sameElements(prim.args(), true)) yield prim;
-            yield new CallTerm.Prim(prim.ref(), prim.ulift(), args);
-          }
-        };
+        if (args.sameElements(prim.args(), true)) yield prim;
+        yield new CallTerm.Prim(prim.ref(), prim.ulift(), args);
```
Comment (imkiva) [inline base/src/main/java/org/aya/core/term/Term.java:126]: What if the args are not reduced but the head is an interval call?

Comment (imkiva) [reply base/src/main/java/org/aya/core/term/Term.java:126]: No.

Comment (ice1000) [reply base/src/main/java/org/aya/core/term/Term.java:126]: Most likely. But how to deal with it?

Comment (imkiva) [reply base/src/main/java/org/aya/core/term/Term.java:126]: I don't think it's here bc unfolding happens in `Expander` after this descent

**Pull request merged**